ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/BlockPropagationManager.java

```diff
@@ -150,27 +149,6 @@ private void onBlockAdded(final BlockAddedEvent blockAddedEvent) {
                   LOG.info("Imported {} pending blocks", r.size());
                 }
               });
-    } else {
-
-      LOG.trace("Not ready for import blocks found for {}", newBlock.getHeader().getNumber());
-
-      pendingBlocksManager
-          .lowestAnnouncedBlock()
-          .map(ProcessableBlockHeader::getNumber)
-          .ifPresent(
-              minAnnouncedBlockNumber -> {
```

Comment:
this, in essence, was a periodic retry mechanism, but it relied on blocks being added.

Reply:
I'm not even sure that adding a new peer will necessarily trigger the search for a new block currently

This mechanism was added as a retry to fix this same problem we have now but it did not work considering what we have on the nodes